corehq/form_processor/tests/test_forms.py

```diff
@@ -81,6 +98,105 @@ def test_get_with_attachments(self):
         self.assertEqual('form.xml', attachment_meta.name)
         self.assertEqual('text/xml', attachment_meta.content_type)
 
+    def test_get_attachment_by_name(self):
```

Comment:
will you delete the tests in `corehq.form_processor.tests.test_form_dbaccessor` in another PR?

Reply:
Yes, along with `FormAccessors` and `FormAccessorSQL`.